.github/workflows/publish_snapshots.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish snapshots to maven
+
+on:
+  workflow_dispatch:
+  push:
+      branches:
+        - main
+        - 1.*
```

Comment:
Will this publish snapshots for like 1.0, 1.1, 1.2 etc? We're only supporting the 1.3 version in the 1.x line so that doesn't seem necessary. Same goes for 2.x where I think the `2.x` branch is all we would need. (Unless I'm misunderstanding what the `*` does here...)

Reply:
You are understanding right! Let me know what should be the trigger values here. I am assuming you want `main`, `1.x` and `2.x`